src/posts/us-exit-documents/index.md

```diff
@@ -10,19 +10,30 @@ tags:
   - data-sovereignty
 ---
 
+{% alert %}
+
+It was brought to my attention that OnlyOffice is _not_ from Latvia, but [actually from Russia with a Latvian shell company](https://dms-solutions.co/blog/dms-solutions-stops-doing-business-with-onlyoffice-due-to-onlyoffice-close-ties-with-russia/).
```

Comment:
...my attention during evaluation/experimentation that...

Reply:
La till en paragraf lite längre ner som täcker det här